README.md

```diff
@@ -1,9 +1,16 @@
 # Scylla monitoring with Grafana and Prometheus
 
-### Notice for users of Scylla version prior to 1.4
+<table style="border-style: solid;">
+  <tr>
```

Comment:
@amnonh please avoid HTML tags in markdown
If you can't get it with simple tables, or quotes, its not worth it.